doctests/home-json.js

```diff
@@ -0,0 +1,149 @@
+// EXAMPLE: js_home_json
+// REMOVE_START
+import assert from 'assert';
+// REMOVE_END
+// STEP_START import
+import {
+    createClient,
+    SchemaFieldTypes,
+    AggregateGroupByReducers,
+    AggregateSteps,
+} from 'redis';
+// STEP_END
+
+// STEP_START connect
+const client = await createClient();
+await client.connect();
+// STEP_END
+// REMOVE_START
+try {
+    await client.ft.dropIndex('idx:users', { DD: true });
+} catch{}
+
+await client.del('user:1', 'user:2', 'user:3');
+// REMOVE_END
+
+// STEP_START create_data
+var user1 = {
+    name: "Paul John",
+    email: "paul.john@example.com",
+    age: 42,
+    city: "London"
+}
+
+var user2 = {
+    name: "Eden Zamir",
+    email: "eden.zamir@example.com",
+    age: 29,
+    city: "Tel Aviv"
+}
+
+var user3 = {
+    name: "Paul Zamir",
+    email: "paul.zamir@example.com",
+    age: 35,
+    city: "Tel Aviv"
+}
+// STEP_END
+
+// STEP_START make_index
+await client.ft.create('idx:users', {
+    '$.name': {
+        type: SchemaFieldTypes.TEXT,
+        AS: 'name'
+    },
+    "$.city": {
+        type: SchemaFieldTypes.TEXT,
+        AS: 'city'
+    },
+    '$.age': {
+        type: SchemaFieldTypes.NUMERIC,
+        AS: 'age'
+    }
+}, {
+    ON: 'JSON',
+    PREFIX: 'user:'
+});
+// STEP_END
+
+// STEP_START add_data
+var user1Added = await client.json.set('user:1', '$', user1);
```

Comment:
there is no reason to make 3 round trips for 3 SET, you can do:
```javascript
const [user1Reply, user2Reply, user3Reply] = await Promise.all([
  client.json.set('user:1', '$', user1),
  client.json.set('user:2', '$', user2),
  client.json.set('user:3', '$', user3)
]);
```

Reply:
Good tip, thanks.